translations/translations.json

```diff
@@ -7093,6 +7094,7 @@
       "settings": "ሴቲንጎች",
       "copyUrl": "አገናኝን ቅዳ",
       "contrastTheme": "የንፅፅር ሁነታ",
+      "contrastThemeDescription": "የንፅፅር ሁነታ የመተግበሪያዎን ተነባቢነት ያሻሽላል።",
```

Comment:
❓ Where do you get all these translations from? We usually don't want to add them ourselves except for our mother languages (+ english). Instead, these should be translated by professional translators later on.

Reply:
Sorry for the effort you put into this and that I did not spot this with the contrast theme. But then please delete the translations again and also those for `contrastTheme` :/

Reply:
yes, I think it is really important to include what you wrote above otherwise it is not much clear..

Reply:
I think it makes it clear, thank you :+1: